x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/TransportSqlQueryAction.java

```diff
@@ -109,4 +121,12 @@ static SqlQueryResponse createResponse(RowSet rowSet, List<ColumnInfo> columns)
                 columns,
                 rows);
     }
+    
+    protected static String username(SecurityContext securityContext) {
```

Comment:
Move this and the method below to a separate package protected. utils class (Transports?).

Reply:
+1